Dockerfile

```diff
@@ -60,13 +60,8 @@ COPY ./runmariadb.sh /bin/runmariadb
 COPY ./configure-ironic.sh /bin/configure-ironic.sh
 COPY ./ironic-common.sh /bin/ironic-common.sh
 
-# TODO(dtantsur): remove these 2 scripts if we decide to
-# stop supporting running all 2 processes via one entry point.
-COPY ./runhealthcheck.sh /bin/runhealthcheck
-COPY ./runironic.sh /bin/runironic
-
 COPY ./dnsmasq.conf.j2 /etc/dnsmasq.conf.j2
 COPY ./inspector.ipxe /tmp/inspector.ipxe
 COPY ./dualboot.ipxe /tmp/dualboot.ipxe
 
-ENTRYPOINT ["/bin/runironic"]
+ENTRYPOINT ["/bin/runironic-api"]
```

Comment:
This script seems to do a lot less than runironic, is  `ironic-dbsync --config-file /etc/ironic/ironic.conf upgrade` not needed? What about cleaning up the logs directory?

Reply:
Realistically, nobody uses this alone. A working installation consists of all 4 entry points. I'm not sure how to handle it, can I just remove `ENTRYPOINT`?

Reply:
Ah you're right, since we change the entrypoint for each container.  That raises another issue though, these containers are launched by the installer, machine-api-operator, and the deployment scripts under baremetal-operator. 

I don't think we can delete that script, it's going to require far too many changes to different places - especially if we have to backport to 4.3. Could we just keep it around but not do the health checks for now?

Reply:
Are you sure we use the old script anywhere? I'm pretty sure we stopped it very early in our development. Using it, for example, means no logs from ironic.